Makefile

```diff
@@ -61,6 +61,16 @@ else
 		gotestsum --format standard-quiet --junitfile test-results/unit-tests.xml -- ./... -p $(GO_PARALLEL_JOBS) -timeout 80s
 endif
 
+test-go-helm-only:
+ifneq ($(CIRCLECI),true)
+		go test -p $(GO_PARALLEL_JOBS) -timeout 80s ./internal/tiltfile -run "(?i)(.*)Helm(.*)"
```

Comment:
How bananas is this? Any other ideas?

Reply:
tbh, we should probably move Helm and its tests to its own plugin, and then a lot of this weirdness would go away